batch2/batch/driver/main.py

```diff
@@ -390,6 +390,29 @@ def validate_int(name, value, predicate, description):
     return web.HTTPFound(deploy_config.external_url('batch2-driver', '/'))
 
 
+@routes.get('/user_resources')
+@web_authenticated_developers_only()
+async def get_user_resources(request, userdata):
+    app = request.app
+    db = app['db']
+
+    sql = '''
+SELECT * from user_resources
+ORDER BY ready_cores_mcpu
+LIMIT 50;
```

Comment:
I think we should do either limit and pagination, or no limit.  I think no limit is fine for now and we can revisit this if/when the page gets slow.